kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/LogReplay.java

```diff
@@ -101,22 +101,20 @@ public static StructType getAddRemoveReadSchema(boolean shouldReadStats) {
 
     private final Path dataPath;
     private final LogSegment logSegment;
-    private final TableClient tableClient;
     private final Tuple2<Protocol, Metadata> protocolAndMetadata;
 
     public LogReplay(
             Path logPath,
             Path dataPath,
             long snapshotVersion,
-            TableClient tableClient,
             LogSegment logSegment,
+            TableClient tableClient,
```

Comment:
Make the `TableClient` as the first argument here and all below functions.

Reply:
Sure, Thanks for the Info.